client/hostvolumemanager/host_volume_plugin.go

```diff
@@ -132,6 +143,10 @@ func NewHostVolumePluginExternal(log hclog.Logger,
 	}, nil
 }
 
+// HostVolumePluginExternal calls an executable on disk. All operations should
+// be idempotent, and safe to be called concurrently per volume ID.
+// For each call, the executable's stdout and stderr may be logged, so plugin
+// authors should not include any sensitive information in their plugin outputs.
```

Comment:
These docstrings are likely to make their way into a spec for plugin authors. We should probably use the term "must" rather than "should" here in that [RFC 2119](https://www.rfc-editor.org/rfc/rfc2119) sense of the terms.

Reply:
>  **MUST**   This word, or the terms "REQUIRED" or "SHALL", mean that the definition is an absolute requirement of the specification.

I don't think this matches, because including secret strings doesn't break anything, and we have no way of enforcing it.

Reply:
Just because it's not enforceable doesn't mean it's compliant with the spec. But "should be idempotent" is probably a better candidate for the stronger "must" wording, because that definitely _will_ break things.